zh_cn/energy.gasturbine.json

```diff
@@ -4,7 +4,7 @@
 
 	"合成:",
 	{"link":"futurepack:lang/part/cr_energy.gasturbine.json"},
-	"通过增加"ram"值以更快的产生和释放能量。",
+	"通过增加/"ram"/值以更快的产生和释放能量。",

```

Comment:
I think you need to use \" instead, or else the JSON format will break.,

Reply:
ok